README.md

```diff
@@ -258,6 +258,22 @@ module "api_gateway" {
 | redeployment_trigger                      | string       | A unique string to force a redeploy of the api gateway. If not set manually, the module will use the configurations for endpoints, api_key, and authorizer config to decide if a redeployment is necessary. | (yes)    |                                                           |
 | tags                                      | map(string)  | Map of tags to assign to the Lambda Function and related resources                                                                                                                                          | no       | `{}`                                                      |
 
+#### setup-tfstate
+
+```hcl
+module "setup_tfstate" {
+  source = "github.com/THEY-Consulting/they-terraform//aws/setup-tfstate"
+
+  prefix = "they"
+  name = "test"
+}
+```
+
+| Variable | Type   | Description                                                     | Required | Default |
+|----------|--------|-----------------------------------------------------------------|----------|---------|
+| name     | string | Name of the app, used for the s3 bucket and dynamoDB table name | yes      |         |
+| prefix   | string | Prefix for the name                                             | yes      |         |
```

Comment:
QoL:

Do we really need two variables to adapt the name? It feels like we should drop name and just use prefix for everything.

We used that naming for `cp-tfstate-arivo-transfer` and `cp-tfstate-mcp-api`, but we also use `tiga-tfstate` and `they-tfstate`. I think the latter two are enough for our requirements, given the nature that one state bucket is enough for any number of workspaces for a given project.

We could just use `cp-arivo-transfer-tfstate` and `cp-mcp-api-tfstate` using only one variable and just hardcoding the `tfstate`

Side note: We should/could probably adapt/migrate the state buckets to have a unique naming scheme.

Reply:
Yeah I was thinking about this as well but opted to be backwards compatible. But you are right, a single variable is much cleaner